_gettingstarted/python-hunter.md

```diff
@@ -0,0 +1,160 @@
+---
+layout: article
+title: Python Hunter
+description: Hunter is a flexible code tracing toolkit.
+tags: [python, hunter, tracing, debugging, logging, inspection]
+image: /assets/images/getting-started/getting-started-python-hunter-cover.png
+cover: /assets/images/getting-started/python-hunter.png
+article_header:
+  type: overlay
+  theme: dark
+  background_image:
+    gradient: 'linear-gradient(135deg, #2BF29F, #FFFF00)'
+date: 2024-02-14
+author:
+  name: Lavanya Mylapalli
+---
+
+## Introduction
+Python Hunter is a versatile code tracing toolkit developed for debugging, logging, inspection, It is not meant for measuring the  coverage. It offers a simple Python API, a comfortable terminal API, and a command-line tool for attaching to processes. If you have any bugs in your code, and if you worry how to trace them, this is the best package that can help you. It will help you in tracing and understanding the flow of your code.Debugging the code is very challenging especially for beginners, it will behave as a debugging assistant that can help you in reavealing the hidden paths of your program that happens during execution.
+
+ **Installation**
+To install python hunter into your project, open your terminal and run the below command.
+
+   ```python
+   pip install hunter
+   ```
+
+**Importing the Trace**
```

Comment:
don't use ** for headings,, instead use ## or ###

** is to make the text bold, ### and ## is for headings